Cargo.toml

```diff
@@ -22,6 +22,7 @@ eyre = "0.6.8"
 
 [features]
 export-abi = ["stylus-sdk/export-abi"]
+debug = ["stylus-sdk/debug"]
 
```

Comment:
I don't think we want this on by default, right? I didn't see console logs used anywhere.

Reply:
Agreed!

Reply:
This doesn't actually enable them - it just gives us the option to enable them if desired